cdap-unit-test/src/test/java/co/cask/cdap/security/AuthorizationTest.java

```diff
@@ -387,11 +380,9 @@ public void testArtifacts() throws Exception {
       authorizer.listPrivileges(ALICE)
     );
     // artifact deployment in this namespace should now succeed, and alice should have ALL privileges on the artifacts
-    NamespacedArtifactId appArtifactId = new NamespacedArtifactId(
-      AUTH_NAMESPACE.getNamespace(), appArtifact.getName(), appArtifact.getVersion().getVersion());
+    ArtifactId appArtifactId = new ArtifactId(AUTH_NAMESPACE.getNamespace(), "app-artifact", "1.1.1");
```

Comment:
oh i removed that because it had to use the fully qualified class name so the lines got really long.  I can make these variables though